.github/workflows/integration_tests.yml

```diff
@@ -14,7 +14,7 @@ on:
         required: true
       apis:
         description: 'CSV of apis to build and test'
-        default: 'admob,analytics,auth,database,dynamic_links,firestore,functions,installations,messaging,remote_config,storage'
+        default: 'admob,analytics,auth,database,dynamic_links,firestore,functions,gma,installations,messaging,remote_config,storage'
```

Comment:
Maybe remove admob from this list for now?

Reply:
Hm, yeah, missed that one.  Then again, it's going right back in presently.